app/models/registration.rb

```diff
@@ -386,6 +386,19 @@ def events_to_associated_events(events)
     end
   end
 
+  def permit_user_cancellation?
+    case competition.competitor_can_cancel
```

Comment:
Could you either do `.to_sym` and check symbols (like `where :always`) or do `to_s` please?
Just want to make sure we don't run into unexpected behavir because in Rails `"foo" == :foo` evaluates to `false`.